src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 server.port=8080
+server.error.include-message=always
```

Comment:
do we need that?

Reply:
I had a problem with JSON error response: it didn't contain the message field. I found in the documentation that the error message and any binding errors are not included in the error page by default and that it can be added and controled by developer using this property.